plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -40,6 +40,8 @@
     private Optional<String> catalogId = Optional.empty();
     private int partitionSegments = 5;
     private int getPartitionThreads = 20;
+    private int readStatisticsThreads = 1;
```

Comment:
Do we need those configuration parameters? Do we expect that we would need to bump it? If not I would rather not have those. The less config params the better.

Reply:
I agree with having as few params as possible but this parameters are useful for performance tuning when working with tables with an high amount of columns, also making the concurrency configurable should help users to deal with the AWS Glue rate limits.